Source/Cogworks.Essentials/Services/MemoryCacheService.cs

```diff
@@ -1,13 +1,18 @@
 ﻿using System;
+using System.Collections.Concurrent;
+using System.Threading;
+using System.Threading.Tasks;
+using Cogworks.Essentials.Constants;
+using Cogworks.Essentials.Services.Interfaces;
 using Microsoft.Extensions.Caching.Memory;
 
 namespace Cogworks.Essentials.Services
 {
     public class MemoryCacheService : ICacheService
     {
-        private static MemoryCacheOptions Options => new MemoryCacheOptions();
+        private static MemoryCache Cache => new MemoryCache(new MemoryCacheOptions());
```

Comment:
Have some idea, I think it would be good to introduce ctors that can handle memory stuff.

```
private readonly IMemoryCache _cache;

public MemoryCacheService(IMemoryCache inMemoryCache)
  => _cache = inMemoryCache;

public MemoryCacheService()
  => _cache = new MemoryCache(new MemoryCacheOptions());
```

This will give some possibility to mock this service in tests and customize `MemoryCache` if we want to change options.

Reply:
First ctor for custom implementation, second as default.